package.json

```diff
@@ -137,9 +137,51 @@
             "Numpy style Python documentation format."
           ],
           "description": "Python docstring format"
+        },
+        "trelent.autodoc.mode": {
+          "type": "string",
+          "default": "Enable Globally",
+          "enum": [
+            "Enable Per-Function",
+            "Enable Globally",
+            "Maintain Docstrings"
+          ],
+          "enumDescriptions": [
+            "If set to 'Enable Per-Function', the auto-doc service will only notify of changes for functions with the tag @trelent-highlight.",
+            "If set to 'Enable Globally', the auto-doc service will be enabled for all functions, functions titled with the tag @trelent-ignore will not be highlighted.",
+            "If set to 'Maintain Docstrings', the auto-doc service will automatically document all functions automatically"
+          ],
+          "description": "Defines the behaviour of the auto-doc service."
```

Comment:
Going to update the descriptions here to be a bit more clear, and have both mode and changeThreshold under trelent.autodoc.

Reply:
Agreed!